### PR TITLE
[DOCS] Updates Quickstart guide

### PR DESCRIPTION
**Pull request by Rachel-Reverie**
## Description
- updates python version
- updates prerequisites to use dynamic python version snippet
- updates related documentation to reference consolidated pages

## Definition of done
- [X] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [X] Appropriate tests and docs have been updated

### CODE REVIEW

File: docs/docusaurus/docs/components/_data.jsx [hunk 1]
```diff
@@ -1,5 +1,5 @@
 export default {
   release_version: 'great_expectations, version 0.17.5',
   min_python: '3.7',
```
Comment (Kilo59) [inline docs/docusaurus/docs/components/_data.jsx]:
```suggestion
  min_python: '3.8',
```

**Review by Kilo59 — CHANGES_REQUESTED** [review]
Need to change the min python version to `3.8`

File: docs/docusaurus/docs/tutorials/quickstart/quickstart.md [hunk 2]
```diff
@@ -38,9 +38,9 @@ Windows support for the open source Python version of GX is currently unavailabl
     pip install great_expectations
     ```
 
-    It can take several minutes for the installation to complete. Jupyter Notebook is included with Great Expectations, and it lets you edit code and view the results of code runs.
+    It can take several minutes for the installation to complete.
 
-2. Open Jupyter Notebook or Terminal and then run the following command to import the `great_expectations` module:
+2. In Python run the following command to import the `great_expectations` module:
```
Comment (kwcanuck) [inline docs/docusaurus/docs/tutorials/quickstart/quickstart.md]:
```suggestion
2. Run the following Python code to import the `great_expectations` module:
```

File: docs/docusaurus/docs/components/_data.jsx (hunk 1 above)
Comment (kwcanuck) [reply docs/docusaurus/docs/components/_data.jsx]: @Kilo59 the Python version content was updated in a previous PR. Here is how the `_data.jsx` file currently appears:

image

The version information is correct in the [latest version of the documentation](https://docs.greatexpectations.io/docs/tutorials/quickstart/):

image

**Review by Kilo59 — APPROVED** [review]

**Pull request merged**